application-templates/typescript/service/src/connector/post-deploy.ts

```diff
@@ -0,0 +1,31 @@
+import { createApiRoot } from '../client/create.client';
+import { assertError, assertString } from '../utils/assert.utils';
+import {
+  createCustomCartDiscountType,
+  createCartUpdateExtension,
+} from './actions';
+
+const CONNECT_APPLICATION_URL_KEY = 'CONNECT_SERVICE_URL';
```

Comment:
Shouldn't we add access to the variable on `process.env` instead?

```suggestion
const CONNECT_APPLICATION_URL_KEY = process.env.CONNECT_SERVICE_URL;
```